module/sell.py

```diff
@@ -44,10 +44,10 @@ def sell(update: Update, context: CallbackContext) -> None:
                     df = get_on_sale()
                     new_row = pd.DataFrame({'ISBN':str(isbn),'Titolo':str(title), 'Autori':str(authors), 'Venditore':str(username), 'Prezzo':str(prezzo)}, index=[len(df)])
                     df = pd.concat([df, new_row])
-                    df.to_csv('data.csv')
+                    df.to_csv('data/on_sale.csv')
                     new_book = pd.DataFrame({'ISBN':str(isbn),'Titolo':str(title), 'Autori':str(authors)}, index=[len(books)])
                     books = pd.concat([books, new_book])
-                    books.to_csv('books.csv')
+                    books.to_csv('data/books_db.csv')
```

Comment:
what about creating a CONST variable with all these strings? you are repeating different time `data/books_db.csv`, if you encapsulate it as variable will be easy to change, next time if you'll need to change this path, you'll need to change only two variables, one for books and one for on_sale.

Let's put it in CONST variable in a separate file (ex. constats.py or shared.py)

Reply:
Should I put it in modules? Also, should I add any function in this file? Like the three getters functions maybe?

Reply:
not sure if I got what you mean, I think that in the future would be better to convert this into a sqlite database, so do not overcharge yourself on this, make a small change like adding a constant in this way
```suggestion
# shared file
BOOKS_DB_PATH = 'data/books_db.csv'
....
# file 1
books.to_csv(BOOKS_DB_PATH)
...
# file 2
books.to_csv(BOOKS_DB_PATH)
```

Reply:
Ok, got it. I was thinking about moving all the getters (like `get_books()`, `get_on_sale()` and `find_book()`) inside the shared.py file. I don't know if it could be of any use though, or even if it would be a best practice or not

Reply:
it depends 🤷‍♂️

Reply:
Anyway, I did the changes and added the shared.py in the last commit 👍🏻